main.cpp

```diff
@@ -55,13 +59,38 @@ video: Chapter 2 - Part 3
 void variableDeclarations()
 {
     //example:
-    int number = 2; //declaration of a variable named "number", that uses the primitive type 'int', and the variable's initial value is '2'
-    
-
-    
-    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
-}
-
+    int number = 2; //declaration of a variable named "number", that uses the primitive type 'int', and the variable's initial value is '2'
+
+    int rainDrops = 45;    
+    int cows = 100;
+    int planets = 8;
+
+    float numSamplesToRender = 56000.0f;
+    float decibelsToGain = 0.5f;
+    float ageOfMySonInYears = 3.5f;
+
+    double lightYearsToAlienLife = 123670978.454;
+    double slicesOfCakeLeftInMyHouse = 0.0002;
+    double percentageOfWheelsOnBrokenCar = 0.75;
+
+    bool rainToday = true;
+    bool rainedLastFriday = false;
+    bool rainTomorrow = 1;
+
+    char PlantInitial = 'O';
+    char mySurnameInitial = 'P';
+    char avgSchoolGrade = 'B';
+
+    unsigned int semiTonesInOctave = 12;
+    unsigned int sumTwoRandomValues = 24 + 10;
+    unsigned int major7 = 11;
+
+    
+    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+
+    ignoreUnused(rainDrops, cows, planets, numSamplesToRender, decibelsToGain, ageOfMySonInYears, lightYearsToAlienLife, slicesOfCakeLeftInMyHouse, percentageOfWheelsOnBrokenCar,  rainToday, rainedLastFriday, rainTomorrow, PlantInitial, mySurnameInitial, avgSchoolGrade, semiTonesInOctave, sumTwoRandomValues, major7);

```

Comment:
include `number` when passing all of your variables to `ignoreUnused()`
- [ ] revise